app/frontend/wallet/blockchain-explorer.js

```diff
@@ -120,6 +127,13 @@ const blockchainExplorer = (CARDANOLITE_CONFIG, walletState) => {
         'Content-Type': 'application/json',
       }
     )
+
+    if (!response.Right) {
+      debugLog(`Unexpected tx submission response: ${response}`)
```

Comment:
Testing for the presence of "response.Left" does not neccesarily cover all the possible bad responses. In theory it should, but the only think I think it is ok to rely on is that everything is ok if the response is wrapped in the "Right" key, otherwise there is an unexpected problem.